### PR TITLE
Update md5s for 2.5.2 CR2

### PR DESCRIPTION
**Pull request by OwenCorrigan76**
Update md5s for 2.5.2 CR2

### CODE REVIEW

File: kafka/modules/kafka/3.4.0/module.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ envs:
     value: "amqstreams-kafka-34-container"
 
 artifacts:
-  - md5: 11c4eb691ae1965b39b7137b894d75ab
+  - md5: 15dcea9f299b4ba03256e85b222acf31
```
Comment (ppatierno) [inline kafka/modules/kafka/3.4.0/module.yaml:11]: this is wrong you got the MD5 for the pom file not the zip. It should be `7891c4a71a2727906f60cf1cf071c4b0`

File: operator/modules/operator/module.yaml [hunk 2]
```diff
@@ -10,17 +10,17 @@ envs:
...
+  - md5: 38b3eeda1cb01a0e9ef2306863673fbb
     name: user-operator-dist.zip
-  - md5: df4498f43627247e6a28e3e6db1abe1f
+  - md5: 3dee382e78e3f3ddb2446e4eaba2a3c5
     name: kafka-init-dist.zip
-  - md5: 487e8e8423e504fd4f3881baf8176650
+  - md5: 00a76af7a07d1f3be367b79d95624a91
     name: strimzi-licenses.tar.gz
-  - md5: 27fb3b7d4831914bad48528b98f4ece1
+  - md5: a3a4ed85fc80291b1ce954f092f08430
```
Comment (ppatierno) [inline operator/modules/operator/module.yaml:23]: this is wrong, you have got the strimzi-kafka-scripts MD5 (which is used for the Kafka image module above). You need the MD% for the strimzi-operator-scripts, it should be `6ea00c042dc6c16e69c94720dac4dbc7`

**Review by ppatierno — CHANGES_REQUESTED** [review]

File: kafka/modules/kafka/3.4.0/module.yaml (hunk 1 above)
Comment (OwenCorrigan76) [reply kafka/modules/kafka/3.4.0/module.yaml]: Fixed

File: operator/modules/operator/module.yaml (hunk 2 above)
Comment (OwenCorrigan76) [reply operator/modules/operator/module.yaml]: Fixed

**Review by ppatierno — APPROVED** [review]

**Pull request merged**